### PR TITLE
Environment Summary implementation

### PR DESCRIPTION
**Pull request by areyesd14**
This PR contains environment summary implementation.

### CODE REVIEW

File: love/src/components/EnvironmentSummary/Cartoons/AuxTelescope.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  const auxtelMouseLeave = () => {
+    setShowAuxtelSummary(false);
+    showIconTemperature();
+  };
+
+  return (
+    <svg viewBox="0 0 444.43 436.59" className={className} {...props}>
+      {/* <text x={135} y={10} width="60" height="40" className={styles.temperature}>
+        10.84 °C
+      </text> */}
```
Comment (sebastian-aranda) [inline love/src/components/EnvironmentSummary/Cartoons/AuxTelescope.jsx:32]: These lines should be removed.

File: love/src/components/EnvironmentSummary/Cartoons/AuxTelescope.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        <rect className={styles.cls2} x="169.41" y="132.63" width="176.93" height="45" />
+        <path
+          className={showAuxtelSummary ? styles.hover : styles.cls5}
+          d="m257.87,1h0c48.83,0,88.46,39.64,88.46,88.46v43.17h-176.93v-43.17c0-48.83,39.64-88.46,88.46-88.46Z"
+        />
+      </g>
+      <g id="AuxTel">
+        {/* <text x={187} y={50} width="60" height="40" className={styles.temperature}>
+          10.36 °C
+        </text> */}
```
Comment (sebastian-aranda) [inline love/src/components/EnvironmentSummary/Cartoons/AuxTelescope.jsx:43]: These lines should be removed.

File: love/src/components/EnvironmentSummary/Cartoons/SimonyiTelescope.jsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  const simonyiOnMouseLeave = () => {
+    setShowSimonyiSummary(false);
+    showIconTemperature();
+  };
+
+  return (
+    <svg viewBox="0 0 726.72 436.7" className={className} {...props}>
+      {/* <text x={245} y={10} width="60" height="40" className={styles.temperature}>
+        10.12°
+      </text> */}
```
Comment (sebastian-aranda) [inline love/src/components/EnvironmentSummary/Cartoons/SimonyiTelescope.jsx:32]: These lines should be removed.

File: love/src/components/EnvironmentSummary/Cartoons/SimonyiTelescope.jsx [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+      <g id="Simonyi">
+        <polygon
+          id="SimonyiDome"
+          className={showSimonyiSummary ? styles.hover : styles.cls6}
+          points="275.31 175.1 275.31 49.4 202.98 1.11 74.29 45.61 20.96 129.31 20.96 176.07 275.31 175.1"
+        />
+        <g id="TMA" className={styles.hoverable} onMouseEnter={simonyiOnMouseEnter} onMouseLeave={simonyiOnMouseLeave}>
+          {/* <text x={185} y={40} width="60" height="40" className={styles.temperature}>
+            18.36 °C
+          </text> */}
```
Comment (sebastian-aranda) [inline love/src/components/EnvironmentSummary/Cartoons/SimonyiTelescope.jsx:42]: These lines should be removed.

File: love/src/components/EnvironmentSummary/Cartoons/WindDirection.jsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+export class WindDirection extends Component {
+  static propTypes = {};
+
+  render() {
+    // TODO: calcular con escala
+    const minSpeed = 0;
+    const maxSpeed = 100;
+    const currentSpeed = 50;
+    const maxArrowHeight = 500;
+    const arrowHeight = (currentSpeed / maxSpeed) * maxArrowHeight;
```
Comment (sebastian-aranda) [inline love/src/components/EnvironmentSummary/Cartoons/WindDirection.jsx:15]: I think this is missing to connect the respective telemetry.

File: love/src/components/EnvironmentSummary/Skymap/Skymap.jsx [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+import SimonyiPointing from '../Pointings/SimonyiPointing';
+import SunPointing from '../Pointings/SunPointing';
+import MoonPointing from '../Pointings/MoonPointing';
+import SunSummary from '../SummaryInformation/SunSummary';
+import MoonSummary from '../SummaryInformation/MoonSummary';
+
+export default class Skymap extends Component {
+  // static propTypes = {
+  //   prop: PropTypes
+  // };
```
Comment (sebastian-aranda) [inline love/src/components/EnvironmentSummary/Skymap/Skymap.jsx:16]: These lines should be removed.

File: love/src/components/EnvironmentSummary/Skymap/Skymap.jsx [hunk 7]
```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react';
+// import PropTypes from 'prop-types';
```
Comment (sebastian-aranda) [inline love/src/components/EnvironmentSummary/Skymap/Skymap.jsx:2]: this line should be removed.

File: love/src/components/EnvironmentSummary/Skymap/SkymapGrid.jsx [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+      <svg
+        className={styles.grid}
+        xmlns="http://www.w3.org/2000/svg"
+        width={this.props.width}
+        height={this.props.width}
+        viewBox={`-${viewboxMargin} -${viewboxMargin} ${w + 2 * viewboxMargin} ${h + 2 * viewboxMargin}`}
+      >
+        <rect className={styles.backgroundRect} width="100%" height="100%" fill="none" />
+        <circle className={styles.backgroundCircle} cx={w / 2} cy={h / 2} r={w / 2 - margin} />
+        {/* <circle className={styles.domeCircle} cx={w / 2} cy={h / 2} r={w / 2 + viewboxMargin - 5} /> */}
```
Comment (sebastian-aranda) [inline love/src/components/EnvironmentSummary/Skymap/SkymapGrid.jsx:159]: This line should be removed.

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
Great job @areyesd14 !! This looks really good, I've left some minor comments. I've also added a few commits fixing the AT pointing and a few interaction issues with the hovers.

File: love/src/components/EnvironmentSummary/Cartoons/AuxTelescope.jsx (hunk 1 above)
Comment (areyesd14) [reply love/src/components/EnvironmentSummary/Cartoons/AuxTelescope.jsx]: Resolved. Thanks!

File: love/src/components/EnvironmentSummary/Cartoons/AuxTelescope.jsx (hunk 2 above)
Comment (areyesd14) [reply love/src/components/EnvironmentSummary/Cartoons/AuxTelescope.jsx]: Resolved. Thanks!

File: love/src/components/EnvironmentSummary/Cartoons/SimonyiTelescope.jsx (hunk 3 above)
Comment (areyesd14) [reply love/src/components/EnvironmentSummary/Cartoons/SimonyiTelescope.jsx]: Resolved. Thanks!

File: love/src/components/EnvironmentSummary/Cartoons/SimonyiTelescope.jsx (hunk 4 above)
Comment (areyesd14) [reply love/src/components/EnvironmentSummary/Cartoons/SimonyiTelescope.jsx]: Resolved. Thanks!

File: love/src/components/EnvironmentSummary/Cartoons/WindDirection.jsx (hunk 5 above)
Comment (areyesd14) [reply love/src/components/EnvironmentSummary/Cartoons/WindDirection.jsx]: You're right! Thanks!! Resolved!

File: love/src/components/EnvironmentSummary/Skymap/Skymap.jsx (hunk 6 above)
Comment (areyesd14) [reply love/src/components/EnvironmentSummary/Skymap/Skymap.jsx]: Resolved. Thanks!

File: love/src/components/EnvironmentSummary/Skymap/Skymap.jsx (hunk 7 above)
Comment (areyesd14) [reply love/src/components/EnvironmentSummary/Skymap/Skymap.jsx]: Resolved. Thanks!

File: love/src/components/EnvironmentSummary/Skymap/SkymapGrid.jsx (hunk 8 above)
Comment (areyesd14) [reply love/src/components/EnvironmentSummary/Skymap/SkymapGrid.jsx]: Resolved. Thanks!

**Review by sebastian-aranda — APPROVED** [review]
Ready to be merged, thanks @areyesd14 !

**Pull request merged**